src/sentry/event_manager.py

```diff
@@ -157,11 +154,6 @@
 # Timeout for cached group crash report counts
 CRASH_REPORT_TIMEOUT = 24 * 3600  # one day
 
-issue_rate_limiter = RedisSlidingWindowRateLimiter(
-    **settings.SENTRY_PERFORMANCE_ISSUES_RATE_LIMITER_OPTIONS
-)
-PERFORMANCE_ISSUE_QUOTA = Quota(3600, 60, 5)
-
```

Comment:
just checking, what is the new way to updating noise config for issue types?

Reply:
It's on the `GroupType` itself, like https://github.com/getsentry/sentry/blob/cc839dd8ab4ebb1b75742f509c531da54b52946b/src/sentry/issues/grouptype.py#L273